src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -256,12 +252,15 @@ private EnumerateSnapshotAccessRequest convertToEnumerateModel(
     return enumerateModel;
   }
 
-  record ParentQueryResult(int parentId, int childId, String childName, boolean hasChildren) {
+  record ParentQueryResult(
+      int parentId, int childId, String childName, long code, int count, boolean hasChildren) {
     ParentQueryResult(ResultSet rs) throws SQLException {
       this(
           rs.getInt(HierarchyQueryBuilder.PARENT_ID),
           rs.getInt(HierarchyQueryBuilder.CONCEPT_ID),
           rs.getString(HierarchyQueryBuilder.CONCEPT_NAME),
+          rs.getLong(HierarchyQueryBuilder.CONCEPT_CODE),
```

Comment:
It looks like the concept code is a string not a long?

Reply:
That's a good point. All concept codes for the concepts in our test data are longs, but the data type in the OMOP CDM is `varchar(50)`. See https://ohdsi.github.io/CommonDataModel/cdm54.html#concept

I guess to be forward compatible this should use `string` instead of `long`, I'll change the code to do this.